src/main/java/org/intellij/sdk/codesync/clients/JSONResponse.java

```diff
@@ -17,10 +17,17 @@
 */
 public class JSONResponse {
     private final int statusCode;
+    private int customErrorCode;
```

Comment:
I do not this one being used anywhere. I think we should not make this a property.

Reply:
We need this to store value of custom error code 
![image](https://github.com/codesyncapp/codesync-intellij/assets/15974895/0ceaef19-cf26-41ee-83c1-c91a993bd6e4)

Reply:
I think that can be converted to a local variable.